models/repo_test.go

```diff
@@ -68,3 +73,27 @@ func TestRepo(t *testing.T) {
 		})
 	})
 }
+
+func TestGetRepositoryCount(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
+	count, _ := GetRepositoryCount(&User{ID: int64(10)})
```

Comment:
Could we assert that the returned `error` is `nil`, instead of just ignoring it?